shared_model/interfaces/signable.hpp

```diff
@@ -34,10 +34,10 @@ namespace shared_model {
     class Signable : public Hashable<Model, OldModel> {
      public:
       /// Type of transaction signature
-      using SignatureType = Signature;
+      using SignatureType = detail::PolymorphicWrapper<Signature>;
 
       /// Type of set of signatures
-      using SignatureSetType = std::unordered_set<const SignatureType>;
+      using SignatureSetType = std::unordered_set<SignatureType>;
```

Comment:
Also, there is need a note about the reason why not available `const` on value type in `unordered_set`.